packages/runtime/container-runtime/src/summary/summaryGenerator.ts

```diff
@@ -273,20 +272,12 @@ export class SummaryGenerator {
 			submitFailureResult?: SubmitSummaryFailureData,
 			nackSummaryResult?: INackSummaryResult,
 		) => {
-			// Report any failure as an error unless it was due to cancellation (like "disconnected" error)
-			// If failure happened on upload, we may not yet realized that socket disconnected, so check
-			// offlineError too.
-			const category =
-				cancellationToken.cancelled || error?.errorType === DriverErrorTypes.offlineError
-					? "generic"
-					: "error";
-
 			const reason = getFailMessage(errorCode);
 			summarizeEvent.cancel(
 				{
 					...properties,
 					reason,
-					category,
```

Comment:
My two cents are that any non-driver error should be an error. It should also be treated as an error.

Reply:
Feel free to resolve